### PR TITLE
mediacheck: Add support for Cloud 6 deps ISO

### PR DESCRIPTION
**Pull request by vuntz**
~~Depends on https://github.com/SUSE/cloud/pull/101 to fully work.~~

### CODE REVIEW

File: scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml [hunk 1]
```diff
@@ -63,6 +64,11 @@
               export RPM_DEPCHECK_MEDIADEPS=" "
               export SRCRPM_DEPCHECK_MEDIADEPS=" "
               export INSTALLCHECK_EXTRA_OPTS="--exclude pattern:"
+          elif [[ "$project" =~ Devel:Cloud:(.*):Appliances ]]; then
+              cloud_version=${BASH_REMATCH[1]}
+              export RPM_DEPCHECK_MEDIADEPS="solv/cloud-$cloud_version-dvd.solv solv/cloud-$cloud_version-updates.solv"
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml]: hmm, why do you add cloud_verison-dvd.solv as a dep? that doesn't make sense to me. the deps iso should not contain anything that depends on cloud 6..

Comment (vuntz) [inline scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml]: I have the SES packages in there, but without the packages that are also in Cloud -- so these packages actually depend on Cloud packages.

But I'm thinking of just putting all packages we need (including Cloud ones) in the deps iso and only use this as the one true source of packages that replaces everything for the appliance.

**Comment by vuntz** [comment]
Ok, so I went ahead and moved the cloud packages to the deps iso, so we don't use anything else (not even the cloud iso). For reference: old commit was dff3983.

File: scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml]: is it actually possible to combine SES with Cloud?

Comment (vuntz) [inline scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml]: > is it actually possible to combine SES with Cloud? I thought there were a few python packages in different versions needed.

There's one conflict (django), which means we have to sacrifice calamari. But that's reasonable for the appliance, I think.

> main downside is that we're loosing the extra skelcds and the license information I guess...

Hrm. Might be an issue, es...

**Comment by dirkmueller** [comment]
I don't think its a good idea to create just one media.. I can't discuss the use case publically though.

**Comment by vuntz** [comment]
Btw, regardless of having all packages in one ISO or not: this patch is needed :-)

**Comment by dirkmueller** [comment]
@vuntz agreed.

+1

**Pull request merged**